src/main/kubectl.ts

```diff
@@ -266,54 +278,51 @@ export class Kubectl {
   }
 
   protected async writeInitScripts() {
+    const kubectlPath = userStore.preferences?.downloadKubectlBinaries ? this.dirname : path.dirname(this.getPathFromPreferences())
     const helmPath = helmCli.getBinaryDir()
     const fsPromises = fs.promises;
     const bashScriptPath = path.join(this.dirname, '.bash_set_path')
-    const bashScriptIsLatest = await this.scriptIsLatest(bashScriptPath)
-    if (!bashScriptIsLatest) {
-      let bashScript = "" + initScriptVersionString
-      bashScript += "tempkubeconfig=\"$KUBECONFIG\"\n"
-      bashScript += "test -f \"/etc/profile\" && . \"/etc/profile\"\n"
-      bashScript += "if test -f \"$HOME/.bash_profile\"; then\n"
-      bashScript += "  . \"$HOME/.bash_profile\"\n"
-      bashScript += "elif test -f \"$HOME/.bash_login\"; then\n"
-      bashScript += "  . \"$HOME/.bash_login\"\n"
-      bashScript += "elif test -f \"$HOME/.profile\"; then\n"
-      bashScript += "  . \"$HOME/.profile\"\n"
-      bashScript += "fi\n"
-      bashScript += `export PATH="${this.dirname}:${helmPath}:$PATH"\n`
-      bashScript += "export KUBECONFIG=\"$tempkubeconfig\"\n"
-      bashScript += "unset tempkubeconfig\n"
-      await fsPromises.writeFile(bashScriptPath, bashScript.toString(), { mode: 0o644 })
-    }
+
+    let bashScript = "" + initScriptVersionString
+    bashScript += "tempkubeconfig=\"$KUBECONFIG\"\n"
+    bashScript += "test -f \"/etc/profile\" && . \"/etc/profile\"\n"
+    bashScript += "if test -f \"$HOME/.bash_profile\"; then\n"
+    bashScript += "  . \"$HOME/.bash_profile\"\n"
+    bashScript += "elif test -f \"$HOME/.bash_login\"; then\n"
+    bashScript += "  . \"$HOME/.bash_login\"\n"
+    bashScript += "elif test -f \"$HOME/.profile\"; then\n"
+    bashScript += "  . \"$HOME/.profile\"\n"
+    bashScript += "fi\n"
+    bashScript += `export PATH="${helmPath}:${kubectlPath}:$PATH"\n`
+    bashScript += "export KUBECONFIG=\"$tempkubeconfig\"\n"
+    bashScript += "unset tempkubeconfig\n"
+    await fsPromises.writeFile(bashScriptPath, bashScript.toString(), { mode: 0o644 })
 
     const zshScriptPath = path.join(this.dirname, '.zlogin')
-    const zshScriptIsLatest = await this.scriptIsLatest(zshScriptPath)
-    if (!zshScriptIsLatest) {
```

Comment:
Why did we remove this check? (and is `scriptIsLatest` function now dead code?)

Reply:
We cannot rely on script version, since user can change kubectl path in preferences. So we need to make sure that kubectl path is correct when running the terminal. Any other ways to solve this?

Reply:
Ok, makes sense. We then probably should delete that function/code so that it won't haunt us.